assets/clients/clients.yaml

```diff
@@ -589,6 +589,17 @@ clients:
       - type: shield
         label: Microsoft Store
         url: https://apps.microsoft.com/detail/9NBZ2BXD4WFZ
+  
+  - name: Jellyfin for Tizen
+    targets: [Tizen]
+    oss: https://github.com/jellyfin/jellyfin-tizen
+    official: true
+    downloads:
+    - type: github
+      owner: jellyfin
+      repo: jellyfin-tizen
```

Comment:
It does not look like it has GitHub releases:
![image](https://github.com/user-attachments/assets/f7acf735-789b-4681-870f-94cc015c8ea8)

So I think we can just remove this section or add a text like "build it yourself"